### PR TITLE
pgcli v1.5.0

### PR DESCRIPTION
**Pull request by j-bennet**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/pgcli.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Pgcli < Formula
   desc "CLI for Postgres with auto-completion and syntax highlighting"
   homepage "http://pgcli.com/"
-  url "https://files.pythonhosted.org/packages/75/68/0aabfa3bdfa6858247712526c8da13ea2fcd86af3d84cf16d7e937f7f971/pgcli-1.3.1.tar.gz"
-  sha256 "49fcfeb60ca0793f060c7c9dcee34c9dc52486092f96853b882de7f7e75c9209"
+  url "https://files.pythonhosted.org/packages/b3/b9/664956a1da5cdab993aafa128087f65acf9975eaca5ee6242c73bde71ec3/pgcli-1.5.0.tar.gz"
+  sha256 "9511d7c9350a400918ee74d5647345f677bf5feea61e8ae631ed2053372f468c"
   revision 1
```
Comment (fxcoudert) [inline Formula/pgcli.rb]: Remove `revision` line

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/pgcli.rb (hunk 1 above)
Comment (fxcoudert) [inline Formula/pgcli.rb]: Otherwise it looks good!

**Comment by j-bennet** [comment]
Removed revision line... I'm not sure how to fix this mess of all the other commits now. :-/

**Comment by ilovezfs** [comment]
git fetch origin
git reset origin/master
git add -p
git commit
git push -f

**Comment by j-bennet** [comment]
@ilovezfs Genius! :) Thanks a lot.

**Comment by ilovezfs** [comment]
you're welcome

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @j-bennet for your contribution!

**Pull request closed without merging**